client/src/rr/dnssec/key_format.rs

```diff
@@ -18,6 +22,8 @@ pub enum KeyFormat {
     Pem,
     /// Raw bytes unformatted
     Raw,
```

Comment:
> Do you plan on supporting password protected pkcs8, btw?

I want to have some solution for the problem that password-protected PKCS#8 keys solve. I'm not sure that password-protected PKCS#8 is the best solution to that for a variety of reasons. I hope to write a blog post about it "soon".